src/app.js

```diff
@@ -153,3 +163,16 @@ app.on('activate', () => {
 app.on('before-quit', () => {
   isQuitting = true
 })
+
+function addCustomCss(windowElement) {
+  fs.readFile(
+    path.join(__dirname, '..', 'static', 'style.css'),
+    'utf-8',
+    (error, data) => {
+      if (!error) {
+        const formattedData = data.replace(/\s{2,10}/g, ' ').trim()
+        windowElement.webContents.insertCSS(formattedData)
+      }
+    }
+  )
+}
```

Comment:
Although it's good that you thought of minifying the CSS, imo there is actual no real benefit of doing it and the CSS is loaded from the local disk anyway.

I'd try to keep it as simple as possible like so:
```js
webContents.insertCSS(readFileSync(path.join(__dirname, '..', 'css', 'style.css'), 'utf8'));
```

Reply:
Done